scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -697,7 +697,25 @@ def log_event(self, event):
 
 def _convert_json_array_to_object(x):
     if isinstance(x, list):
-        return {str(i): _convert_json_array_to_object(x[i]) for i in range(len(x))}
+        # Special handling of @Name fields per customer request:
+        # Ie [{"@Name:":"n1", "a1":"v1", ...}, ...] becomes {"n1":{"a1":"v1", ...}, ...}
+        # Duplicate @Names are handled by appending the index number if possible.
+        # Otherwise the (stringified) index number becomes the field name.
+        rv = {}
+        for i in range(len(x)):
```

Comment:
It seems that the `x[i]` is not changing during the whole iteration, so I think we can use something like
`for i, x_e in enumerate(x):` to avoid extra dict lookups, but I guess, that's more a matter of taste rather than real issue :)

Reply:
Ah good point, will start using enumerate.  (I would change this code but trying to get this release out quickly)